analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -190,34 +191,36 @@ public void Verify_UnchangedFiles(string unchangedFileName, bool expectedProtobu
 #if NET
 
         [TestMethod]
-        public void Verify_Razor() =>
+        public void Verify_Razor()
+        {
+            using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+
             // Currently only the symbols from .cs files are computed since .razor support is not yet implemented.
             CreateBuilder(ProjectType.Product, "Razor.razor", "ToDo.cs")
                 .WithConcurrentAnalysis(false)
                 .VerifyUtilityAnalyzer<SymbolReferenceInfo>(symbols =>
-                {
-                    symbols.Should().ContainSingle();
-                    symbols[0].FilePath.Should().EndWith("ToDo.cs");
-                    symbols[0].Reference
-                              .Select(x => x.Declaration)
-                              .Should()
-                              .BeEquivalentTo(new[]
-                                  {
-                                      new TextRange { StartLine = 3, EndLine = 3, StartOffset = 13, EndOffset = 17 },
-                                      new TextRange { StartLine = 5, EndLine = 5, StartOffset = 19, EndOffset = 24 },
-                                      new TextRange { StartLine = 6, EndLine = 6, StartOffset = 16, EndOffset = 22 }
-                                  });
-                });
+                    {
+                        var orderedSymbols = symbols.OrderBy(x => x.FilePath).ToArray();
```

Comment:
Just a general remark on the fact that `OrderBy` on string follows the local culture. 
Not a problem in practice here I guess (as far as I know) because strings sorted are under our control.